src/mumble/MainWindow.cpp

```diff
@@ -564,6 +562,14 @@ void MainWindow::updateTransmitModeComboBox() {
 	}
 }
 
+QMenu *MainWindow::createPopupMenu() {
+	if (g.s.wlWindowLayout == Settings::LayoutCustom) {
+		return QMainWindow::createPopupMenu();
+	}
+
+	return new QMenu();
```

Comment:
I'm fine with returning NULL, but from my reading:

"Ownership of the popup menu is transferred to the caller"

means the caller is responsible for this not leaking?